File: average_numbers.py
import string

def get_user_nums():
    user_in = ''
    nums = []
    while user_in != 'done':
        user_in = input('enter a number, or "done" to get average: ')
        if user_in.isdigit():
            nums.append(int(user_in))
    return nums

def get_lst_avg(a_list):
    avg = round(sum(a_list)/len(a_list), 2)
    return avg


def main():
    lst_num = get_user_nums()
    print('average: {}'.format(get_lst_avg(lst_num)))


if __name__ == "__main__":
   main()

File: javascript/js_seed/js/average_numbers.js
function get_user_nums() {
    let user_in;
    let nums = [];
    while (user_in != 'done') {
        user_in = prompt('enter a number, or "done" to get average: ').toLowerCase();
        if (user_in === 'done') {
            console.log(get_avg(nums))
        } else {
            nums.push(parseInt(user_in));
            console.log(nums)
        }

    }
}

function get_avg(arr) {
    function add(a, b) {
        return a + b;
    }
    let sum = arr.reduce(add, 0);
    return sum / arr.length
}

get_user_nums();